### PR TITLE
docs: update README with setup instructions

### LINKED ISSUE #25: Update README with how to setup and run flutter instructions.
## Description
As a developer,,
I need to make sure contributors can setup and run flutter on their local machine so they can start contributing to the project,

## Mocks
N/A

## Acceptance Criteria

- [ ] Update README.md with setup instructions

## Definition of Done
- [ ] All of the required items are completed.
- [ ] Approval by 1 mentor.

## Estimation
1 hour

### PR DESCRIPTION
**Pull request by mtreacy002**
### Description
Include a summary of the change and relevant motivation/context. List any dependencies that are required for this change.

Fixes #25 and replace [PR#34](https://github.com/anitab-org/mentorship-flutter/pull/34)

### Flutter Channel:
- [x] I have used the Flutter Beta channel on my local machine 

### Type of Change:
- Documentation

### How Has This Been Tested?
- running flutter app on VSCode IDE

### Checklist:
- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials
- [x] I have made corresponding changes to the documentation

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -48,6 +48,27 @@ Occasional UI bugs may occur – in this case, please create an issue.
 - To save JWT token, [Flutter Secure Storage](https://pub.dev/packages/flutter_secure_storage) is used
 - If you find something in code that looks a bit odd, it might be some useful extension method from `lib/extensions`
 
+#### Setup instructions
```
Comment (isabelcosta) [inline README.md:51]:
```suggestion
### Setup instructions
```

**Review by bartekpacia — APPROVED** [review]

**Review by isabelcosta — APPROVED** [review]
I guess the .gitignore is part of a successful setup :P 
This looks great and I am glad you got to learn a bit about flutter through this issue/PR @mtreacy002 
Thank you for this contribution 🎉

**Comment by isabelcosta** [comment]
why is the travis PR failing 🙃

**Comment by techno-disaster** [comment]
> why is the travis PR failing upside_down_face

That looks like a error on travis side, i have rerun the test lets see

**Comment by techno-disaster** [comment]
@isabelcosta yep, build was failing due to travis. new results [here](https://travis-ci.org/github/anitab-org/mentorship-flutter)

**Pull request merged**